rust/lance-index/src/vector/ivf/transform.rs

```diff
@@ -137,9 +218,31 @@ impl<T: ArrowFloatType + L2 + Dot + ArrowPrimitiveType> Transformer for IvfTrans
             })?;
 
         let mat = MatrixView::<T>::try_from(fsl)?;
-        let part_ids = self.compute_partitions(&mat).await;
-        let field = Field::new(PART_ID_COLUMN, part_ids.data_type().clone(), true);
-        Ok(batch.try_with_column(field, Arc::new(part_ids))?)
+        let part_ids = self.compute_multiple_partitions(&mat).await;
```

Comment:
hmm, maybe we should use a different transformer for search?

Reply:
do we use transformer for search?
the code path should be `IVFIndex::find_partitions -> IvfImpl::find_partitions -> kmeans::find_partitions` right?